docs/interpreter/spark.md

```diff
@@ -212,28 +213,32 @@ Once `SPARK_HOME` is set in `conf/zeppelin-env.sh`, Zeppelin uses `spark-submit`
 
 Here are few examples:
 
-* SPARK\_SUBMIT\_OPTIONS in conf/zeppelin-env.sh
+* `SPARK_SUBMIT_OPTIONS` in `conf/zeppelin-env.sh`
 
+  ```bash
     export SPARK_SUBMIT_OPTIONS="--packages com.databricks:spark-csv_2.10:1.2.0 --jars /path/mylib1.jar,/path/mylib2.jar --files /path/mylib1.py,/path/mylib2.zip,/path/mylib3.egg"
+  ```
+    
+* `SPARK_HOME/conf/spark-defaults.conf`
 
-* SPARK_HOME/conf/spark-defaults.conf
-
+  ```
     spark.jars        /path/mylib1.jar,/path/mylib2.jar
     spark.jars.packages   com.databricks:spark-csv_2.10:1.2.0
     spark.files       /path/mylib1.py,/path/mylib2.egg,/path/mylib3.zip
+  ```
 
 ### 3. Dynamic Dependency Loading via %spark.dep interpreter
 > Note: `%spark.dep` interpreter is deprecated since v0.6.0.
-`%spark.dep` interpreter load libraries to `%spark` and `%spark.pyspark` but not to  `%spark.sql` interpreter so we recommend you to use first option instead.
+`%spark.dep` interpreter loads libraries to `%spark` and `%spark.pyspark` but not to  `%spark.sql` interpreter. So we recommend you to use the first option instead.
 
```

Comment:
Is it true that `%spark.dep` don't load library to `%spark.sql` ? Because as my understanding, they share the same SparkContext, so should share the libraries. \cc @minahlee to confirm